server/index.js

```diff
@@ -1,6 +1,21 @@
+const cluster = require('cluster');
+const numCPUs = require('os').cpus().length;
+if (cluster.isMaster) {
+  console.log(`Master ${process.pid} is running`);
+  
+  // Fork workers.
+  for (let i = 0; i < numCPUs; i++) {
+    cluster.fork();
+  }
+  
+  cluster.on('exit', (worker, code, signal) => {
+    console.log(`worker ${worker.process.pid} died`);
+  });
+  return;
+}
+
```

Comment:
Can I be a little curious about what you use the cluster for?

Reply:
Sure, Node is single threaded, so by default it's using one CPU core. But using the cluster module you can utilize all the cores on your computer. There is a master process that will distribute the requests to all child processes (in my case 8), so you can have 8 Express instances running at the same time. 
An EC2 has one core tho, so you can't really use this after deployment, I was just experimenting.